src/DynamoCore/Models/DynamoModel.cs

```diff
@@ -1733,6 +1760,36 @@ private void OnWorkspacePropertyChanged(object sender, PropertyChangedEventArgs
                 OnPropertyChanged("RunEnabled");
         }
 
+        /// <summary>
+        /// Use the renderPackageFactoryAssembly tag in the DynamoCore.dll.cfg file to find an
+        /// IRenderPackageFactory implementation.
+        /// </summary>
+        private void SetRenderPackageFactory(string renderPackageFactoryAsm)
+        {
+            var asmPath =
+                Path.Combine(
+                    Path.GetDirectoryName(Assembly.GetExecutingAssembly().Location),
+                    renderPackageFactoryAsm);
+
+            if (!File.Exists(asmPath))
+            {
+                throw new Exception("The specified render package factory assembly does not exist.");
+            }
+
+            var asm = Assembly.LoadFrom(asmPath);
+
+            var factoryType = asm.GetTypes().FirstOrDefault(t =>typeof(IRenderPackageFactory).IsAssignableFrom(t));
+            if (factoryType == null)
+            {
+                throw new Exception("An implementation of IRenderPackageFactory could not be found.");
+            }
+
+            // Construct the factory using the default constructor.
+            var factoryConstructor = factoryType.GetConstructor(System.Type.EmptyTypes);
+            RenderPackageFactory = (IRenderPackageFactory)factoryConstructor.Invoke(null);
+            RenderPackageFactory.MaxTessellationDivisions = MaxTessellationDivisions;
+        }
```

Comment:
I disagree that this needs to "come in the back door".  When you're doing something like this, it indicates that you have a "special circumstance" that requires you to subvert the compiler.  We don't have such a case here.